src/main/tls.c

```diff
@@ -4233,6 +4237,26 @@ SSL_CTX *tls_init_ctx(fr_tls_server_conf_t *conf, int client, char const *chain_
 		}
 	}
 
+#if OPENSSL_VERSION_NUMBER >= 0x10101000L
+	/*
+	 * SSL_CTX_set1_sigalgs_list() macro is missing const modifier for string argument
+	 * thus we have to make a copy of const char *sigalgs_list prior calling the macro.
+	 */
+	if (conf->sigalgs_list) {
+		char *sigalgs_list_copy = talloc_strdup(conf, conf->sigalgs_list);
```

Comment:
You could just use `memcpy` from one pointer to another.  :)  See `/* const issues */` all over the server code.